### ISSUE TITLE
Failed to get comic metadata

### ISSUE THREAD

**Issue by Ju1810** [Sorrow446/MUR_Go#3]
Some comics can't be downloaded, MUR_Go display the following error:
Failed to get comic metadata. json: cannot unmarshal array into Go struct field .data.results.issue_meta.purchase_data of type struct { DigitalPrice float64 "json:\"digital_price\""; StoreURLMobile string "json:\"store_url_mobile\"" }

**Comment by Sorrow446** [comment]
Yeah. I can't fix it because I don't have a subscription anymore.

**Comment by somekrank23** [comment]
Comics that get this error are the ones that (for some reason) have the price missing on Marvel Unlimited's database. I have added a modification to the Structs.go file that fixes this issue.
